homeassistant/components/youless/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The youless integration."""
+import asyncio
+from urllib.error import URLError
+
+import voluptuous as vol
+from youless_api import YoulessAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+
+from ...exceptions import PlatformNotReady
```

Comment:
Only use relative paths within the current component. In this case, use a full import.